public/app/features/logs/logsFrame.test.ts

```diff
@@ -107,11 +107,6 @@ describe('parseLogsFrame should parse different logs-dataframe formats', () => {
     ]);
 
     const result = parseLogsFrame({
-      meta: {
-        custom: {
-          frameType: 'LabeledTimeValues',
-        },
-      },
```

Comment:
One remaining question for me is, if we should keep one test with that frametype because Loki is still generating this frametype? And then once we remove it from Loki, we could also remove the test? I am good with either way though.

Reply:
Good idea, I brought back the test in https://github.com/grafana/grafana/pull/77806/commits/05c7901d410c1faf0cd53ef19f09df96be9620f6